deps/build.jl

```diff
@@ -21,7 +20,7 @@ if !isfile(target) || !isfile(tagfile) || readchomp(tagfile) != "$vers $(Sys.WOR
         run(unpack_cmd(tarball, ".", ".gz", ".tar"))
         cd(srcdir) do
             println("Compiling libblosc...")
-            run(`$make -f ../../make.blosc LIB=../../$target`)
+            run(`$MAKE_CMD -f ../../make.blosc LIB=../../$target`)
```

Comment:
Note ver 0.6 of BinDeps takes care of  the `gmake` vs `make`  command for `bsd` and non-`bsd` setups